instrumentation/google.golang.org/grpc/otelgrpc/stats_handler.go

```diff
@@ -172,15 +280,25 @@ func handleRPC(ctx context.Context, rs stats.RPCStats) {
 				semconv.MessageUncompressedSizeKey.Int(rs.Length),
 			),
 		)
+	case *stats.OutTrailer:
 	case *stats.End:
+		var rpcStatusAttr attribute.KeyValue
+
 		if rs.Error != nil {
 			s, _ := status.FromError(rs.Error)
 			span.SetStatus(codes.Error, s.Message())
-			span.SetAttributes(statusCodeAttr(s.Code()))
+			rpcStatusAttr = semconv.RPCGRPCStatusCodeKey.Int(int(s.Code()))
 		} else {
-			span.SetAttributes(statusCodeAttr(grpc_codes.OK))
+			rpcStatusAttr = semconv.RPCGRPCStatusCodeKey.Int(int(grpc_codes.OK))
 		}
+		span.SetAttributes(rpcStatusAttr)
 		span.End()
+
+		metricAttrs = append(metricAttrs, rpcStatusAttr)
+		r.rpcDuration.Record(context.TODO(), float64(rs.EndTime.Sub(rs.BeginTime)), metric.WithAttributes(metricAttrs...))
+		r.rpcRequestsPerRPC.Record(context.TODO(), gctx.messagesReceived, metric.WithAttributes(metricAttrs...))
+		r.rpcResponsesPerRPC.Record(context.TODO(), gctx.messagesSent, metric.WithAttributes(metricAttrs...))
```

Comment:
This would be a proper solution: https://github.com/open-telemetry/opentelemetry-go/pull/4671#issuecomment-1783534940

Reply:
Ok, I will change like this.

Reply:
Done @pellared